docker/config/application.yaml

```diff
@@ -43,6 +43,9 @@ fhirdata:
   rowGroupSizeForParquetFiles: 33554432   # 32mb
   viewDefinitionsDir: "config/views"
   sinkDbConfigPath: "config/hapi-postgres-config_local_views.json"
+  fhirSinkPath: "http://sink-server:8080/fhir"
```

Comment:
Please add these to [pipelines/controller/config/application.yaml](https://github.com/google/fhir-data-pipes/blob/master/pipelines/controller/config/application.yaml) too and for each of them add some documentation.

Reply:
done